src/main/java/com/microsoft/sqlserver/jdbc/tdsparser.java

```diff
@@ -23,10 +23,11 @@
      * token handler says to stop by returning false from one of the token handling methods.
      */
     static void parse(TDSReader tdsReader, String logContext) throws SQLServerException {
-        parse(tdsReader, new TDSTokenHandler(logContext));
+        parse(tdsReader, new TDSTokenHandler(logContext), false);
     }
 
-    static void parse(TDSReader tdsReader, TDSTokenHandler tdsTokenHandler) throws SQLServerException {
+    static void parse(TDSReader tdsReader, TDSTokenHandler tdsTokenHandler,
+            boolean readOnlyWarningsFlag) throws SQLServerException {
```

Comment:
The javadoc for `parse(...)` should be updated to have the new parameter documented here. Took a bit of skimming through the code to understand what it does.

Reply:
Added.